api_fhir/converters/test_locationConverter.py

```diff
@@ -0,0 +1,92 @@
+
+
+class LocationConverterTestCase(TestCase):
+
+    __TEST_ID = 1
+    __TEST_HF_CODE = "12345678"
+    __TEST_HF_NAME = "TEST_NAME"
+    __TEST_HF_LEVEL = "H"
+    __TEST_ADDRESS = "TEST_ADDRESS"
+    __TEST_PHONE = "133-996-476"
+    __TEST_FAX = "1-408-999 8888"
+    __TEST_EMAIL = "TEST@TEST.com"
+
+    def test_to_fhir_obj(self):
+        imis_hf = self.__create_imis_health_facility_test_instance()
+        fhir_location = LocationConverter.to_fhir_obj(imis_hf)
+
+        self.assertEqual(2, len(fhir_location.identifier))
+        for identifier in fhir_location.identifier:
+            if identifier.get("type").get("code") == Stu3IdentifierConfig.get_fhir_id_type_code():
+                self.assertEqual(imis_hf.id, identifier.get("value"))
+            elif identifier.get("type").get("code") == Stu3IdentifierConfig.get_fhir_facility_id_type():
+                self.assertEqual(imis_hf.code, identifier.get("value"))
+        self.assertEqual(imis_hf.name, fhir_location.name)
+        self.assertEqual(Stu3LocationConfig.get_fhir_code_for_hospital(), fhir_location.type.get('coding')[0]
+                         .get('code'))
+        self.assertEqual(imis_hf.address, fhir_location.address.get("text"))
+        self.assertEqual(3, len(fhir_location.telecom))
+        for telecom in fhir_location.telecom:
+            if telecom.get("system") == ContactPointSystem.PHONE.value:
+                self.assertEqual(imis_hf.phone, telecom.get("value"))
+            elif telecom.get("system") == ContactPointSystem.EMAIL.value:
+                self.assertEqual(imis_hf.email, telecom.get("value"))
+            elif telecom.get("system") == ContactPointSystem.FAX.value:
+                self.assertEqual(imis_hf.fax, telecom.get("value"))
+
+    def test_to_imis_obj(self):
+        fhir_loaction = self.__create_fhir_location_test_instance()
+        imis_hf = LocationConverter.to_imis_obj(fhir_loaction, None)
+
+        self.assertEqual(self.__TEST_HF_CODE, imis_hf.code)
+        self.assertEqual(self.__TEST_HF_NAME, imis_hf.name)
+        self.assertEqual(self.__TEST_HF_LEVEL, imis_hf.level)
+        self.assertEqual(self.__TEST_ADDRESS, imis_hf.address)
+        self.assertEqual(self.__TEST_PHONE, imis_hf.phone)
+        self.assertEqual(self.__TEST_FAX, imis_hf.fax)
+        self.assertEqual(self.__TEST_EMAIL, imis_hf.email)
+
+    def __create_imis_health_facility_test_instance(self):
+        hf = HealthFacility()
+        hf.id = self.__TEST_ID
+        hf.code = self.__TEST_HF_CODE
+        hf.name = self.__TEST_HF_NAME
+        hf.level = self.__TEST_HF_LEVEL
+        hf.address = self.__TEST_ADDRESS
+        hf.phone = self.__TEST_PHONE
+        hf.fax = self.__TEST_FAX
+        hf.email = self.__TEST_EMAIL
+        return hf
+
+    def __create_fhir_location_test_instance(self):
+        location = Location()
+        identifiers = [LocationConverter.build_fhir_identifier(self.__TEST_HF_CODE,
```

Comment:
Change `identifiers` to `indentifier`  without `[]` and next `location.identifier = [identifier]`

Reply:
Corrected. Thanks.